src/cascadia/TerminalApp/ActionArgs.h

```diff
@@ -421,6 +423,62 @@ namespace winrt::TerminalApp::implementation
         }
     };
 
+    struct CloseOtherTabsArgs : public CloseOtherTabsArgsT<CloseOtherTabsArgs>
+    {
+        CloseOtherTabsArgs() = default;
+        GETSET_PROPERTY(uint32_t, Index, 0);
+
+        static constexpr std::string_view IndexKey{ "index" };
+
+    public:
+        hstring GenerateName() const;
+
+        bool Equals(const IActionArgs& other)
+        {
+            auto otherAsUs = other.try_as<CloseOtherTabsArgs>();
+            if (otherAsUs)
+            {
+                return otherAsUs->_Index == _Index;
+            }
+            return false;
+        };
+        static FromJsonResult FromJson(const Json::Value& json)
+        {
+            // LOAD BEARING: Not using make_self here _will_ break you in the future!

```

Comment:
Why?

Reply:
The time to ask that was when these were all added 😉 

```
src\cascadia\TerminalApp\ActionArgs.h
78:            // LOAD BEARING: Not using make_self here _will_ break you in the future!
110:            // LOAD BEARING: Not using make_self here _will_ break you in the future!
136:            // LOAD BEARING: Not using make_self here _will_ break you in the future!
164:            // LOAD BEARING: Not using make_self here _will_ break you in the future!
192:            // LOAD BEARING: Not using make_self here _will_ break you in the future!
227:            // LOAD BEARING: Not using make_self here _will_ break you in the future!
262:            // LOAD BEARING: Not using make_self here _will_ break you in the future!
296:            // LOAD BEARING: Not using make_self here _will_ break you in the future!
326:            // LOAD BEARING: Not using make_self here _will_ break you in the future!
354:            // LOAD BEARING: Not using make_self here _will_ break you in the future!
385:            // LOAD BEARING: Not using make_self here _will_ break you in the future!
413:            // LOAD BEARING: Not using make_self here _will_ break you in the future!
```

Reply:
Oh that's just copypasta from all the other ActionArgs. I hit that the first time I ever did that, and it was a pain to debug, so I left that comment there for future reference. It's lived in the body of the `FromJson` functions every since.

Reply:
Hah ok.